src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -413,6 +429,37 @@ class ExplorerButtonGroup extends React.Component {
     });
   }
 
+  // This is a generic way to export a PFB to a third party, deprecating
+  // exportToTerra and exportToSevenBridges.
+  // See docs/export-pfb-to-url.md
+  // Code flow (it's confusing):
+  // 1. User clicks a 'export-pfb-to-url' button
+  // 2. Store target URL as state.targetURLTemplate
+  // 2. Kick off a PFB export job with this.exportToPFB()
+  // 3. componentDidUpdate polls the exportToPFB job status, when complete it will call
+  //    sendPFBToURL(this.state.targetURLTemplate)
+  exportPFBToURL = (targetURLTemplate) => () => {
```

Comment:
I hope since this is general enough, that we can remove the support and related codes/configs for `exportingToTerra` and `exportingToSevenBridges` and use this general solution instead

It will make our code and config a lot cleaner. And I think BDCat is the only project that is deploying those 2 buttons into prod, so it shouldn't break a lot of stuff 😅

Reply:
Yes, we can replace the exportToTerra and exportToSevenBridges code/configs with this new feature! BDC and Anvil both use the export to Terra / SBG feature in prod.